experimental/generation/generator/packages/library/templates/standard/datetimeV2Entity-add-date.dialog.lg

```diff
@@ -9,26 +9,26 @@
     "$schema": "${appSchema}",
     "$kind": "Microsoft.OnAssignEntity",
     "operation": "Add(${property})",
-    "entity": "datetime",
+    "entity": "datetimeV2",
     "actions": [
         {
         "$kind": "Microsoft.IfCondition",
         "condition": "@datetime.type != 'date'",
         "actions": [
             {
                 "$kind": "Microsoft.SendActivity",
-                "activity": "\${getDateValidationText()}"
+                "activity": "\${${property}_invalidDate(@datetime)}"
```

Comment:
I think all the @datetime should be changed to @datetimeV2

Reply:
For English v2 is preferred.